composer/loggers/mlflow_logger.py

```diff
@@ -160,7 +160,10 @@ def __init__(
                         conda_channel='conda-forge',
                     ) from e
                 databricks_username = WorkspaceClient().current_user.me().user_name or ''
-                self.experiment_name = '/' + os.path.join('Users', databricks_username, self.experiment_name)
+                if self.experiment_name.startswith('/Shared/'):
+                    self.experiment_name = os.path.join('Users', databricks_username, self.experiment_name)
+                else:
+                    self.experiment_name = '/' + os.path.join('Users', databricks_username, self.experiment_name)
```

Comment:
Curious if the solution might also work with:
```suggestion
                self.experiment_name = os.path.join('Users', databricks_username, self.experiment_name)
```

Reply:
@j316chuck @mvpatel2000 tried this out and this doesn't work because databricks requires an absolute path, starting with an '/' - i think there are future design choices we can iron out for how we want to interpret absolute paths vs relative paths to folders within /Users/, but this temporarily works as a bandaid solution to enable users who just want to input an absolute path to /Shared/ instead of /Users/ (which, as of now, is the only other folder under workspaces and will automatically be treated as a subfolder under Users if prepended wit the extra '/')